iati/core/tests/test_rulesets.py

```diff
@@ -237,3 +238,118 @@ def rule_type(self):
     @pytest.fixture
     def valid_case(self):
         return {'paths': ['path_1', 'path_2']}
+
+    @pytest.fixture
+    def invalid_cases(self):
+        return {}
+
+
+class TestRuleDependent(RuleSubclassTestBase):
+    """A container for tests relating to RuleDependent."""
+
+    @pytest.fixture
+    def rule_type(self):
+        return 'dependent'
+
+    @pytest.fixture
+    def valid_case(self):
+        return {'paths': ['path_1', 'path_2']}
+
+    @pytest.fixture
+    def invalid_cases(self):
+        return {}
+
+
+class TestRuleSum(RuleSubclassTestBase):
+    """A container for tests relating to RuleSum."""
+
+    @pytest.fixture
+    def rule_type(self):
+        return 'sum'
+
+    @pytest.fixture
+    def valid_case(self):
+        return {'paths': ['path_1', 'path_2'], 'sum': 3}
+
+    @pytest.fixture(params=[{'paths': ['path_1', 'path_2']}, {'sum': 100}, {}])
```

Comment:
Suggest adding a comment of some kind to indicate what each test case is testing.